### PR TITLE
Fastapi version

### PR DESCRIPTION
**Pull request by yash561**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+$ git switch -c remotes/origin/fastapi_version
+```
+
+## Setup
+
+```bash
+# Create service container
+$ docker build --tag fastapi_ynlp .
+$ docker run --detach --name fastapi_backend --publish 80:80 fastapi_ynlp
+
```
Comment (pandevim) [inline README.md]: missed ``` here

File: README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# Returns the top k comments for video specified in data.json.
+$  curl "http://localhost/score/comments" --request POST --header "Content-Type: application/json" --data @data.json
+```
+
+```bash
+# Try a post request on the transcript endpoint.
+# Returns video transcript for specified video.
+$ curl "http://localhost/score/transcripts" --request POST --header "Content-Type: application/json" --data @data.json
+```
+![screenshot](output.png)
```
Comment (pandevim) [inline README.md:35]: Screenshots should only be mentioned in the PR description.
```suggestion

```

Comment (pandevim) [reply README.md:35]: Also [output.png](https://github.com/YouTubeNLP/Backend/blob/fastapi_version/output.png) file should be deleted.

**Comment by yash561** [comment]
As I am using tiangolo/uvicorn-gunicorn-fastapi image we don't need to do CMD["python","main.py"] and EXPOSE 80 as this both things are done automatically.
output

**Pull request merged**